eventlet/green/ssl.py

```diff
@@ -40,59 +53,56 @@ class GreenSSLSocket(_original_sslsocket):
     settimeout(), and to close/reopen the connection when a timeout
     occurs at an unexpected juncture in the code.
     """
+    def __new__(cls, sock=None, keyfile=None, certfile=None,
+                server_side=False, cert_reqs=CERT_NONE,
+                ssl_version=PROTOCOL_SSLv23, ca_certs=None,
+                do_handshake_on_connect=True, *args, **kw):
+        if sys.version_info < (3, 7):
```

Comment:
suggestion,
all the conditions of `sys.version_info < (3, 7)` can be pre-defined with

`python
is_under_3_7 = sys.version_info < (3, 7)
`
in the global scope

Reply:
Done